insolar/flow/flow.go

```diff
@@ -23,14 +23,20 @@ import (
 )
 
 // Handle is a one-function synchronous process that can call routines to do long processing.
-//
 // IMPORTANT: Asynchronous code is NOT ALLOWED here.
+// To create a new Handle of a given message use handler.NewHandler procedure.
+// After creating a Handle you can register it in MessageBus like this:
+// `h.Bus.MustRegister(insolar.TypeGetObject, createdHandle.WrapBusHandle)`
+// You can find an example in insolar/ladger/artifactmanager/handler.go
 type Handle func(context.Context, Flow) error
 
 // MakeHandle is a function that constructs new Handle.
 type MakeHandle func(bus.Message) Handle
 
 // Procedure is a task that can execute itself.
+// Please not that the Procedure is canceled if a pulse happens during it's execution.
+// Thus if you have multiple steps that can be executed in different pulses split them into separate Procedures.
+// Otherwise join the steps to a single Procedure.
```

Comment:
Otherwise join the steps **in**to a single Procedure.

Reply:
ok